plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotationManager.kt

```diff
@@ -469,6 +519,30 @@ class PointAnnotationManager(
       }
     }
 
+  /**
+   * The TextFont property
+   *
+   * Font stack to use for displaying text.
+   */
+  var textFont: List<String>?
```

Comment:
If it's null, what default value for text font?

Reply:
shall we include the default value here?(maybe it's not required though)